src/Faker/Provider/Image.php

```diff
@@ -91,4 +91,52 @@ public static function image($dir = null, $width = 640, $height = 480, $category
 
         return $fullPath ? $filepath : $filename;
     }
+
+
+
+    /**
+     * Create an image Fake
+     *
+     *
+     * @example '/path/to/dir/13b73edae8443990be1aa8f1a483bc27.jpg'
+     */
+    public static function createImage($dir = null, $width = 640, $height = 480, $fullPath = true, $word = null)
+    {
+        $dir = is_null($dir) ? sys_get_temp_dir() : $dir; // GNU/Linux / OS X / Windows compatible
+        // Validate directory path
+        if (!is_dir($dir) || !is_writable($dir)) {
+            throw new \InvalidArgumentException(sprintf('Cannot write to directory "%s"', $dir));
+        }
+
+        $name = md5(uniqid(empty($_SERVER['SERVER_ADDR']) ? '' : $_SERVER['SERVER_ADDR'], true));
+        $filename = $name .'.jpg';
+        $filepath = $dir . DIRECTORY_SEPARATOR . $filename;
+
+        $word = isset($word) ? $word : "{$width}x{$height}";
+
+        $im = imagecreatetruecolor($width, $height);
```

Comment:
This depends on gd being installed... I'm not a big fan of pushing the base requirements for Faker.